miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -1313,3 +1432,144 @@ end
 export.get_kernel_procedures_ptr
     push.KERNEL_PROCEDURES_PTR
 end
+
+# ACCOUNT DATA LOADERS
+# =================================================================================================
+
+#! Saves storage slots data into memory and validates that the storage commitment matches the 
+#! sequential storage hash.
+#!
+#! Inputs:
+#!   Operand stack: [STORAGE_COMMITMENT]
+#!   Advice map: {
+#!     STORAGE_COMMITMENT: [num_storage_slots, [STORAGE_SLOT_DATA]],
+#!   }
+#! Outputs:
+#!   Operand stack: []
+#! 
+#! Where:
+#! - STORAGE_COMMITMENT is the commitment of the foreign account's storage.
+#! - num_storage_slots is the number of foreign account's storage slots.
+#! - STORAGE_SLOT_DATA is the data contained in the storage slot which is constructed as follows: 
+#!   [SLOT_VALUE, slot_type, 0, 0, 0]
+#!
+#! Panics if:
+#! - the number of account storage slots exceeded the maximum limit of 255.
+#! - the computed account storage commitment does not match the provided account storage commitment
+export.save_account_storage_data
+    # move storage slot data from the advice map to the advice stack
+    adv.push_mapval push.15160 drop                # TODO: remove line, see miden-vm/#1122
+    # OS => [STORAGE_COMMITMENT]
+    # AS => [num_storage_slots, [STORAGE_SLOT_DATA]]
+
+    # push the number of storage slots onto the operand stack before storing it into memory
+    adv_push.1
+    # OS => [num_storage_slots, STORAGE_COMMITMENT]
+    # AS => [[STORAGE_SLOT_DATA]]
+
+    # assert that account does not exceed allowed maximum number of storage slots
+    dup exec.account::get_max_num_storage_slots lt assert.err=ERR_ACCT_TOO_MANY_STORAGE_SLOTS
```

Comment:
Shouldn't this be `lte` instead of `lt`?

Reply:
Yes, I think you are right. And the same in the `save_account_procedure_data`. I'll fix it